tastyworks/models/watchlists.py

```diff
@@ -23,10 +21,18 @@ class WatchlistGroup(object):
     def __init__(self):
         self.watchlists = {}
 
-    async def load_watchlists(self):
-        request_url = '{}/public_watchlists?include_synthetic=true'.format(
-            BASE_URL
+    async def load_watchlists(self, session):
+        """
+        Gets watchlists
+
+        Args:
+            session (TastyAPISession): The session to use.
+        """
+
+        request_url = '{}/public-watchlists'.format(
```

Comment:
Have you given this a test run by the way?

Reply:
I'm not quite sure what you define as a test, but I did test the endpoints with and without a session (it is required).  I have not yet tested the application though.

Reply:
Fair enough. If we're sure that endpoint works then it's all fine by me :)